twine/commands/check.py

```diff
@@ -93,7 +93,7 @@ def _check_file(
     content_type, params = cgi.parse_header(description_content_type)
     renderer = _RENDERERS.get(content_type, _RENDERERS[None])
 
-    if description in {None, "UNKNOWN\n\n\n"}:
+    if description is None or description.rstrip() == "UNKNOWN":
```

Comment:
Looks like the new tests caught an issue where packages built on Windows wouldn't report a missing description, probably because `description` is something like `UNKNOWN\r\n\r\n`.